### PR TITLE
Refactor: rename `Concert` to `ScheduledItem`

### PR DESCRIPTION
**Pull request by zyuiop**
As we discussed wednesday, events can have more activities than just `Concert`. I renamed the class to `ScheduledItem` accordingly. I also added a few useful fields for the next issues.

### CODE REVIEW

**Comment by zyuiop** [comment]
This PR solves #55

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <string name="title_activity_event_picking">EventPickingActivity</string>
     <string name="concert_no_artist">Unknown artist</string>
     <string name="concert_no_genre">Unknown genre</string>
-    <string name="concert_no_description">Sorry ! There is no description for this concert :/</string>
+    <string name="concert_no_description">Sorry ! There is no description for this scheduledItem :/</string>
```
Comment (vquelque) [inline app/src/main/res/values/strings.xml]: Maybe no need to refactor concert to ScheduleItem in the string :)

**Review by vquelque — DISMISSED** [review]

**Review by zbirobin — DISMISSED** [review]
All seems OK !

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (zyuiop) [reply app/src/main/res/values/strings.xml]: Yep indeed.

**Comment by zyuiop** [comment]
Merged in #62

**Pull request closed without merging**